opencti-platform/opencti-graphql/src/modules/entitySetting/entitySetting.graphql

```diff
@@ -27,6 +27,15 @@ type DefaultValueAttribute {
   defaultValues: [DefaultValue!]!
 }
 
+type OverviewLayoutCustomizationParameter {
```

Comment:
I thought we agreed yesterday to do only Front in first chunk ? What changed ?

Reply:
What ? We agreed on passing the mocked config from the backend